api/src/graphql/connectors/index.js

```diff
@@ -287,3 +297,28 @@ export function searchRealities(driver, label, term) {
   `;
   return runQueryAndGetRecords(driver.session(), query, { term });
 }
+
+export function getPeopleTwoStepsFromReality(driver, { nodeId }) {
+  const query = `
+    MATCH (n {nodeId:'${nodeId}'})<-[*0..2]-(p:Person) 
+    WITH collect(distinct p) as pe
+    UNWIND pe as people
+    RETURN people
+  `;
+  return runQueryAndGetRecords(driver.session(), query, { nodeId });
+}
+
+export function getRealityData(driver, { nodeId }) {
```

Comment:
Can we name this function something more specific? Either something that describes what data is fetched or alludes to how it is used (for emails).

Reply:
Changed to getEmailData. Good pointer!